docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -0,0 +1,192 @@
+
+### Event Callbacks
+
+The following callbacks can be configured for the integration in `feedbackIntegration({})` or passed in the `FeedbackForm` component props:
+
+| Callback          | Parameters                                                 | Default behavior       | Description                                                                                                  |
+| ----------------- | ---------------------------------------------------------- | ---------------------- | ------------------------------------------------------------------------------------------------------------ |
+| `onFormOpen`      |                                                            |                        | Callback when form is opened.                                                                                |
+| `onFormClose`     |                                                            | The form is unmounted. | Callback when form is closed and not submitted.                                                              |
+| `onSubmitSuccess` | `data: FeedbackFormData`                                   |                        | Callback when feedback is successfully submitted.                                                            |
+| `onSubmitError`   | `error: Error`                                             |                        | Callback when feedback is unsuccessfully submitted.                                                          |
+| `onFormSubmitted` |                                                            | The form is unmounted. | Callback when the feedback form is submitted successfully, and the SuccessMessage is complete, or dismissed. |
+| `onAddScreenshot` | `attachFile: (filename: string, data: Uint8Array) => void` |                        | Callback when a screenshot is added.                                                                         |
+
+#### Screenshots
+
+The screenshot functionality is disabled by default. To enable it, pass an `imagePicker` integration library or set the `enableScreenshot` option to `true` and implement the `onAddScreenshot` callback.
+
+##### Integrating with an Image Picker Library 
+
+Currently the supported libraries are:
+- [expo-image-picker](https://docs.expo.dev/versions/latest/sdk/imagepicker/) (tested with version `16.0`)
+- [react-native-image-picker](https://github.com/react-native-image-picker/react-native-image-picker) (tested with version `7.2` and `8.0`)
+
+You just need to import the library and pass it to the `feedbackIntegration` method.
+
+```javascript
+import * as ImagePicker from 'expo-image-picker';
+```
+
+```javascript
+import * as ImagePicker from 'react-native-image-picker';
+```
+
+```javascript
+Sentry.init({
+  integrations: [
+    Sentry.feedbackIntegration({
+      imagePicker: ImagePicker,
+    }),
+  ],
+});
+```
```

Comment:
![Screenshot 2025-02-13 at 15 06 48](https://github.com/user-attachments/assets/516356ad-46b1-42e3-b431-e0d7729a44e6)

To display the code blocks under each other, there has to be text in between.

````suggestion
```javascript
import * as ImagePicker from 'expo-image-picker';
```

Or `react-native-image-picker`:

```javascript
import * as ImagePicker from 'react-native-image-picker';
```

And then pass it to `feedbackIntegration`:

```javascript
Sentry.init({
  integrations: [
    Sentry.feedbackIntegration({
      imagePicker: ImagePicker,
    }),
  ],
});
```
````